README.md

```diff
@@ -118,7 +111,7 @@ Currently **189 helpers** in **20 categories**:
 
 ### [array helpers](#array)
 
-Visit the: [code](lib/array.js) | [unit tests](test/array.js) | [issues](https://github.com/helpers/handlebars-helpers/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+array+helpers))
+Visit the: [code](lib/array.js) | [unit tests](test/array.js)
 
 * **[after](#after)** ([code](lib/array.js#L22) | [tests](test/array.js#L13))
```

Comment:
Are these links to the code line numbers (ie, `lib/array.js#13`) still correct, or have they changed since the fork/refactoring?